global-news-app/src/assets/styles/theme.ts

```diff
@@ -0,0 +1,10 @@
+import {  DefaultTheme } from "styled-components";
+
+const colors = {
+  white: "#FFFFFF",
+  black: "black",
```

Comment:
I think it is better to use the same value type, e.g. hex colors for each color instance